launch_testing_ament_cmake/cmake/add_launch_test.cmake

```diff
@@ -122,6 +122,20 @@ function(add_launch_test filename)
     "--package-name=${PROJECT_NAME}"
   )
 
+  if(NOT DEFINED ENV{ROS_DOMAIN_ID})
+    # In the absence of a ROS_DOMAIN_ID, launch_testing will pick a ROS_DOMAIN_ID
+    # that is not in use by another launch_testing instance running on this machine.
+    # This allows tests to run in parallel.  If a ROS_DOMAIN_ID is already set,
```

Comment:
(Might not be relevant if this gets moved but the style guide mandates a new line for each sentence: see https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#markdown-restructured-text-docblocks)

Reply:
A better place for this is in the README.md anyway, I think.  I doubt people would discover it here